test/test_cpp_bin_float.cpp

```diff
@@ -89,24 +93,24 @@ void test_special_cases()
    BOOST_CHECK_EQUAL(max_val * -2, -inf_val);
    BOOST_CHECK_EQUAL(max_val / half, inf_val);
    BOOST_CHECK_EQUAL(max_val / -half, -inf_val);
-   BOOST_CHECK_EQUAL(max_val / min_val, inf_val);
-   BOOST_CHECK_EQUAL(max_val / -min_val, -inf_val);
+   BOOST_CHECK_EQUAL(max_val / denorm_min_val, inf_val);
```

Comment:
I would have both the new *and* old tests here.  Note that the new tests would have passed before (divide by zero).

Reply:
OK